packages/components/src/Avatar/Avatar.stories.mdx

```diff
@@ -0,0 +1,122 @@
+import { useState } from "react";
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Tooltip } from "@jobber/components/Tooltip";
+import { Avatar } from "@jobber/components/Avatar";
+
+<Meta title="Components/Avatar" component={Avatar} />
+
+# Avatar
+
+An avatar will be used to display an visual identifier for an individual user.
+
+```ts
+import { Avatar } from "@jobber/components/Avatar";
+```
+
+<Canvas>
+  <Story name="Avatar">
+    {() => {
+      return (
+        <Avatar
+          imageUrl="https://images.unsplash.com/photo-1533858539156-90ea20bafd17?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1650&q=80"
+          name="The Jobbler"
+          size="large"
+          initials="JBLR"
+          color="var(--color-indigo)"
+        />
+      );
+    }}
+  </Story>
```

Comment:
let's implement the args here to so the `Canvas` works

https://github.com/GetJobber/atlantis/blob/640ebd2d5e126c1a0dc5ab323bfbbab3a000676e/packages/components/src/Button/Button.stories.mdx?plain=1#L23-L29

https://github.com/GetJobber/atlantis/blob/640ebd2d5e126c1a0dc5ab323bfbbab3a000676e/packages/components/src/Button/Button.stories.mdx?plain=1#L30